gulpfile.js

```diff
@@ -27,7 +27,6 @@ plugins.uglify = require('gulp-uglify');
 const paths = {
   src: 'app/assets/',
   dist: 'app/static/',
-  templates: 'app/templates/',
```

Comment:
Just double checking here:  are we sure we don't need this anymore?

Reply:
So leaving these gives the error: 

```[13:35:13] 'images' errored after 2.45 s
[13:35:13] Error: ENOENT: no such file or directory, scandir 'gsa/notifications-admin/undefinedassets/images'
[13:35:13] 'default' errored after 2.45 s
make: *** [bootstrap] Error 1
```

I manually looked for this directory and it no longer exists and running the project locally I see no missing images anywhere. Additionally, vscode suggests that that path doesnt exist either...oh wait, looks what i found...

If you notice, the path `templates` doesn't even match up to `paths.template`, its missing a `s` so it doesn't even work anyways and I still get the same error:

```[13:38:36] 'images' errored after 2.54 s
[13:38:36] Error: ENOENT: no such file or directory, scandir '/Users/ajanousek/gsa/notifications-admin/app/templates/assets/images'
[13:38:36] 'default' errored after 2.54 s
make: *** [bootstrap] Error 1
```

So I am confident that we can remove this code, considering it doesn't even work :)

Reply:
Fair enough, thank you for double checking and looking! :-)